packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js

```diff
@@ -126,14 +125,14 @@ MeetingInfoUtil.isPhoneNumber = (phoneNumber) => {
 };
 
 MeetingInfoUtil.getHydraId = (destination) => {
-  const decodedDestination = deconstructHydraId(destination);
+  const {type, id, cluster} = deconstructHydraId(destination);
 
-  if (decodedDestination && UUID_REG.test(decodedDestination.id)) {
-    if (decodedDestination.type === _ROOM_) {
-      return {room: true, destination: decodedDestination.id};
+  if (id && UUID_REG.test(id)) {
+    if (type === _ROOM_) {
+      return {room: true, destination: id, cluster};
     }
-    if (decodedDestination.type === _PEOPLE_) {
-      return {people: true, destination: decodedDestination.id};
+    if (type === _PEOPLE_) {
+      return {people: true, destination: id, cluster};
```

Comment:
Enhanced this method to report cluster.

Reply:
Can you have this refactoring change in a separate commit?